pkg/internal/apis/acme/types_issuer.go

```diff
@@ -23,27 +23,49 @@ import (
 	cmmeta "github.com/jetstack/cert-manager/pkg/internal/apis/meta"
 )
 
-// ACMEIssuer contains the specification for an ACME issuer
+
+// ACMEIssuer contains the specification for an ACME issuer.
+// This uses the RFC8555 specification to obtain certificates by completing
+// 'challenges' to prove ownership of domain identifiers.
+// Earlier draft versions of the ACME specification are not supported.
```

Comment:
They don't get published, but I opted to do this a) for consistency with other API doc packages, b) because it's not that difficult to copy and paste them over and c) for consistency with upstream APIs 😄 

+1 to something to automate syncing, although there are slight tweaks made (e.g. removing the `kubebuilder` annotations and the `optional` markers etc)